pages/index.tsx

```diff
@@ -99,11 +101,28 @@ export default function Home(props: { users: Array<any> }) {
   };
 
   const Shifts = (props: { shifts: Array<PageObjectResponse> }) => {
+    const [showShmirotOnly, setShowShmirotOnly ] = useState<boolean>(false);
+    let moreThanOneShift=true;
     if (props.shifts.length === 0) {
       return <NoShifts />;
     }
+    if (props.shifts.length === 1) {
+      moreThanOneShift=false;
+    }
     return (
-      <Collapse.Group>
+       <Collapse.Group>
+           { userId && moreThanOneShift && <Grid.Container gap={2} justify="flex-end" >
```

Comment:
if `moreThanOneShift` is only used here as a predicate for conditional rendering, you can simply use `props.shifts.length > 1`

I think it's descriptive enough.

Reply:
change the concept